.github/ISSUE_TEMPLATE/chore.md

```diff
@@ -0,0 +1,22 @@
+---
+name: Chore
+title: "[Chore]: [OVERALL_TASK_TO_DO]"
+about: Use this template to track a chore that needs to be done.
+labels: ["chore"]
+assignees: ""
+---
```

Comment:
Oh, this fills in the corresponding parts of the issue?

Reply:
Yep!

So in this example it doesn't assign a user, but auto-labels the issue with "chore"